api/namespace/applications/models/application.py

```diff
@@ -54,11 +54,11 @@
             description="When the application was last edited",
             example="2022-12-25 00:00:00",
         ),
-        "sections": fields.List(
+        "forms": fields.List(
```

Comment:
its a fair point - it will have knock on effect to the frontnend though (reading object rather than list). Worth saying we control the form added into the database when we mint the application so i believe the user has no way of adding duplicate forms as it stands (they just update the forms already there).

Reply:
in fact after digging this model is only used on GETs so maybe we could rename to reflect this?